### PR TITLE
ProxyCredentials test create

### PR DESCRIPTION
**Pull request by Meylin-V**

### CODE REVIEW

File: src/test/java/executor/service/model/ProxyCredentialsTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    @Test
+    public void getPasswordTest() {
+        assertEquals(PASSWORD, proxyCredentials.getPassword());
+    }
+
+    @Test
+    public void setUsernameTest() {
+        proxyCredentials.setUsername(NEW_USERNAME);
+        assertEquals(NEW_USERNAME, proxyCredentials.getUsername());
+    }
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyCredentialsTest.java:36]: Let's add a negative tests when we expect assertNonEquals for different values

File: src/test/java/executor/service/model/ProxyCredentialsTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    @Test
+    public void equalsTest() {
+        ProxyCredentials pC = new ProxyCredentials(USERNAME, PASSWORD);
+        ProxyCredentials pCAnother = new ProxyCredentials(NEW_USERNAME, NEW_PASSWORD);
+
+        assertEquals(proxyCredentials, pC);
+        assertNotEquals(proxyCredentials, pCAnother);
+        assertNotEquals(proxyCredentials, null);
+    }
```
Comment (maksymivanov2510) [inline src/test/java/executor/service/model/ProxyCredentialsTest.java:61]: Let's add tests for comparing (comparing I mean check 'equals' and ''hashCode' methods) null with empty object or / and  comparing 2 empty objects.

**Comment by Meylin-V** [comment]
All test was added, ready to review

**Review by maksymivanov2510 — APPROVED** [review]
Looks good! Let's merge the branch
